account/src/routes/index.js

```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import usuarios from "./usersRoutes.js"
+
+const routes = (app) => {
+    app.route('/').get((req, res) => {
+        res.status(200).send({titulo: "Ecomm Users"})
+    })
```

Comment:
do you need this route?

Reply:
not really, I just put it on to test and forgot to take it off